src/richie/apps/courses/static/richie/images/catalog_visibility/hidden.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>iko/prof onlysvg</title>
+    <g id="iko/prof-only" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <rect id="Rectangle" x="0" y="0" width="24" height="24"></rect>
+        <path d="M1.65179764,6.64815571 C1.27396888,6.59078843 0.892686768,6.7211929 0.605822599,7.00595187 C0.318932269,7.29071084 0.154374528,7.70207552 0.154374528,8.13451147 C0.154060587,9.63235255 0.154060587,11.556694 0.154060587,11.556694 C0.154060587,15.1850516 2.86607156,18.1260938 6.21216615,18.1260938 C7.44914852,18.1260938 8.59935218,17.7239857 9.55747589,17.03413 C10.3443183,16.4679686 11.0807207,16.0641629 11.8416106,16.0641629 C12.6032068,16.0641629 13.3396093,16.4679686 14.1256929,17.03413 C15.0838428,17.7239857 16.234779,18.1260938 17.4710288,18.1260938 C20.8172542,18.1260938 23.5299716,15.1850516 23.5299716,11.556694 L23.5299716,8.13448179 C23.5299716,7.70195681 23.3651522,7.29047342 23.0784189,7.00565509 C22.7914239,6.72080709 22.4099848,6.59037294 22.0322084,6.6477699 C18.6567602,7.15454887 15.145794,7.32344551 11.8416106,7.32344551 C8.53747952,7.32344551 5.02661789,7.15454887 1.65179764,6.64815571 Z M20.3819218,12.2503868 C20.0548993,11.4761544 18.8756299,10.9021645 17.4710288,10.9021645 C16.0671865,10.9021645 14.888074,11.4761544 14.5610253,12.2503868 C14.888074,13.0253256 16.0671865,13.5993155 17.4710288,13.5993155 C18.8756299,13.5993155 20.0548993,13.0253256 20.3819218,12.2503868 Z M8.99536339,11.9767082 C8.66758216,11.2017694 7.48917608,10.628512 6.0846012,10.628512 C4.68002633,10.628512 3.50162024,11.2017694 3.17457154,11.9767082 C3.50162024,12.7509406 4.68002633,13.3249305 6.0846012,13.3249305 C7.48917608,13.3249305 8.66758216,12.7509406 8.99536339,11.9767082 Z" id="Fill-1" fill="#031963"></path>
+    </g>
+</svg>
```

Comment:
Here an optimized version:

```suggestion
<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M0 0h24v24H0z"/><path d="M1.652 6.648a1.22 1.22 0 0 0-1.046.358 1.593 1.593 0 0 0-.452 1.129v3.422c0 3.628 2.712 6.57 6.058 6.57a5.7 5.7 0 0 0 3.345-1.093c.787-.566 1.524-.97 2.285-.97.761 0 1.498.404 2.284.97a5.701 5.701 0 0 0 3.345 1.092c3.346 0 6.059-2.94 6.059-6.57V8.135c0-.432-.165-.844-.452-1.128a1.22 1.22 0 0 0-1.046-.358c-3.375.507-6.886.675-10.19.675-3.305 0-6.815-.168-10.19-.675Zm18.73 5.602c-.327-.774-1.506-1.348-2.911-1.348-1.404 0-2.583.574-2.91 1.348.327.775 1.506 1.35 2.91 1.35 1.405 0 2.584-.575 2.91-1.35Zm-11.387-.273c-.327-.775-1.506-1.348-2.91-1.348-1.405 0-2.583.573-2.91 1.348.327.774 1.505 1.348 2.91 1.348 1.404 0 2.583-.574 2.91-1.348Z" fill="#031963"/></g></svg>
```

Reply:
Ok.

And also changed the svg's color to the `indianred3` value (`#df484b`). The @sampaccoud  asked this change so we don't include a new color on the site.